content/en/docs/tasks/observability/metrics/lightstep/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Viewing Istio Metrics in Cloud Observability from ServiceNow
+description: This task shows you how to use OpenTelemetry to ingest and view Istio metrics in Cloud Observability (formerly Lightstep).
+weight: 11
```

Comment:
Also the weight here is used to set the order of the links within the page. You can look at your preview at https://deploy-preview-13779--preliminary-istio.netlify.app/latest/docs/tasks/observability/metrics/ to see if the orderings correct. Higher numbers go down the order.